### PR TITLE
Add top-level "results" key

### PR DESCRIPTION
**Pull request by otoolep**
Compiles, but not all tests pass because of inproper marshalling of Results.

### CODE REVIEW

**Comment by otoolep** [comment]
WIP, not ready yet for merging.

**Comment by otoolep** [comment]
Ready to merge except for `Stopper` issue, all tests pass,

File: client/influxdb.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ func NewClient(c Config) (*Client, error) {
 	return &client, nil
 }
 
-func (c *Client) Query(q Query) (influxdb.Results, error) {
+func (c *Client) Query(q Query) (*influxdb.Results, error) {
```
Comment (dgnorton) [inline client/influxdb.go]: Reference types usually aren't returned as pointers.  E.g., http://golang.org/pkg/strings/#Split returns `[]string` not `*[]string`.

Comment (dgnorton) [inline client/influxdb.go]: Scratch that...I see the type changed.

**Comment by otoolep** [comment]
LGTM from @dgnorton -- merging now.

**Pull request merged**